src/app/containers/ATIAnalytics/canonical/index.test.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { create, act } from 'react-test-renderer';
+import * as testingLibrary from '@testing-library/react';
```

Comment:
Consider importing `render` and `act` only  `import { render, act } from '@testing-library/react';`

Reply:
See previous comment about `render` variable already having been imported for enzyme https://github.com/bbc/simorgh/blob/b0a4bf986094bdb3b88300537deb3e54a5cae209/src/app/containers/ATIAnalytics/canonical/index.test.jsx#L2-L3